src/fondue/Fondue.js

```diff
@@ -352,4 +384,8 @@ export default class Fondue {
 		}
 		return null;
 	}
+
+	get customText() {
+		return this._font.opentype.tables.name.get(19) || false;
```

Comment:
Maybe return `null` instead of false, WDYT? `false` would be a good fallback return value if the function would otherwise return a boolean. But since this is returning a string on success it's a bit unexpected to return a boolean IMO.